### ISSUE TITLE
Built in GUI templates?

### ISSUE THREAD

**Issue by dmgeurts** [PowerDNS-Admin/PowerDNS-Admin#1034]
Looking at the css I noticed "skin-blue" is used and there are files using names with other colours. Which made me wonder if there's an easy way to change the colour scheme used?

I'm having to support multiple PowerDNS-admin interfaces, so having a visual reminder as to what system a user is using would be very helpful.

**Comment by RGanor** [comment]
Hey @dmgeurts !
There was a pull request that added custom css feature, i didnt try it yet, but it seems pretty straightforward.
#944

**Comment by dmgeurts** [comment]
Yes. this works really well. Sadly this doesn't allow you to change the theme or 'logo' text. But any CSS can be modified easily this way.

**Comment by RGanor** [comment]
Great! Loved to hear that!
Can i close this issue?

**Comment by dmgeurts** [comment]
For, hiding things in the interface, yes. However is there a way to change the theme? When inspecting the page I can change the theme by editing the in-line code.

How can I do this permanently, as this isn't CSS?

**Comment by dmgeurts** [comment]
![image](https://user-images.githubusercontent.com/2071992/140988161-09e67838-86b2-4713-8922-9b6dd2570900.png)

**Comment by RGanor** [comment]
As far as I can tell, this part is from AdminLTE package and cant be changed in run time.
The good news is that the code is opensource so you can change the HTML here:
https://github.com/ngoduykhanh/PowerDNS-Admin/blob/bf836621088972cfb53813bf1f338ad1fa7c9505/powerdnsadmin/templates/base.html#L35

**Comment by dmgeurts** [comment]
Simply changing the statement in that file to read `skin-red` doesn't change the page header. Am I missing a step like a rebuild?

**Comment by RGanor** [comment]
How do you deploy your application? docker/systemd?

**Comment by dmgeurts** [comment]
I used the instructions from here: https://github.com/ngoduykhanh/PowerDNS-Admin/wiki/Running-PowerDNS-Admin-on-Ubuntu-or-Debian.

**Comment by dmgeurts** [comment]
I resolved the issue with some manual editing:

`/var/www/html/pdns/powerdnsadmin/templates/base.html` I changed the class to 'skin-red'

```
<body class="hold-transition skin-red sidebar-mini {% if not SETTING.get('fullscreen_layout') %}layout-boxed{% endif %}">
```

For me, this was enough to identify this server as different from the other pdns-admin server. I actually also edited the CSS to hide IPv6 and reverse zones as they're irrelevant for this secondary server and would only confuse the guys using the server to check the zones. The custom CSS feature was great for this, so I don't need to worry too much about updates as with the skin colour change.

**For updates I do the following**

First backup the files that were edited:
`mkdir -p ~/pdns-admin.backup/templates` 
`cp /var/www/html/pdns/powerdnsadmin/templates/base.html ~/pdns-admin.backup/templates/base.html` 

Then do git stuff:
`cd /var/www/html/pdns` 
`sudo git status` 

Store the local changes and revert to branch:
`sudo git stash` 

Update the code from the current branch:
`sudo git pull` 

Show differences between the current branch and our last stash:
`sudo git stash show -p stash@{0}` 

If the changes are not to the files I edited (skin-red) for base.html. Then simply reapply the stash, I've not encountered what will happen if I reapply the stash to an updated base.html file. I'll cross that bridge when I get to it:
`sudo git stash pop`
